app/components/agenda/agendaitem/agendaitem-controls/component.js

```diff
@@ -137,15 +150,15 @@ export default Component.extend(isAuthenticatedMixin, {
   },
 
   async deletePostponedPhases(subcase) {
-    const postponedPhases = await subcase.get("postponedPhases");
-    if (postponedPhases && postponedPhases.length) {
-      await Promise.all(
-        postponedPhases.map(phase => {
-          phase.destroyRecord();
-        })
-      );
-    }
-    return subcase;
+      const postponedPhases = await subcase.get("postponedPhases");
+      if (postponedPhases && postponedPhases.length) {
+        await Promise.all(
+          postponedPhases.map(phase => {
+            phase.destroyRecord();
+          })
+        );
+      }
+      return subcase;
```

Comment:
Check format please.

Reply:
Seems ok according to the editor.

Reply:
But it doesn't seem OK to my **EYES** 👀 

Why is there an indentation of 4 spaces here?

Reply:
Fixed it with my editor.